### ISSUE TITLE
Error! Something went wrong while re-provisioning this device.

### ISSUE THREAD

**Issue by dyoga** [bitsy-ai/octoprint-nanny-plugin#105]
![Screenshot 2021-01-31 145236](https://user-images.githubusercontent.com/5736583/106386238-5712d280-63d4-11eb-992b-0446d7deae32.png)

Hello i can't register my printer

Printer
Creality CR-6 SE

Octopi Ver.
1.5.3

Octoprint Nanny Ver.
0.4.1

**Comment by leigh-johnson** [comment]
Hey @dyoga, thanks for the report!

Can you try this again with 0.4.2 and let me know how it goes? 

If that doesn't work for you, attach the logs in ~/.octoprint/logs so I can look into this. Thanks again!

**Comment by dyoga** [comment]
Hello, now the connection was succsesfull.
But my Printer is shown as "idle" with active monitoring :(

> Hey @dyoga, thanks for the report!
> 
> Can you try this again with 0.4.2 and let me know how it goes?
> 
> If that doesn't work for you, please attach the logs in ~/.octoprint/logs so I can look into this. Thanks again!

**Comment by leigh-johnson** [comment]
Woo glad device registration worked for you!

Bummer on the "idle" status, I must have missed a edge case. Can you collect the following info to help me reproduce?

1. Does this happen when you start monitoring in Print Nanny's UI, OctoPrint's UI, or both?
2. Can you start monitoring, sing a short song like "baby shark", stop monitoring, and then upload ~/.octoprint/logs for me to inspect?

Thanks!

**Comment by dyoga** [comment]
Hello, you are very fast :D

Monitoring starts now but no livestream. I'm using Octolapse if it's relevant.

There is no Octoprint Nanny log. which log should i upload? The Octoprint log is about 490MB :o


![Screenshot 2021-02-02 152128](https://user-images.githubusercontent.com/5736583/106613487-88bda200-656a-11eb-8876-0b72ec07108b.png)


> Woo glad device registration worked for you!
> 
> Bummer on the "idle" status, I must have missed a edge case. Can you collect the following info to help me reproduce?
> 
> 1. Does this happen when you start monitoring in Print Nanny's UI, OctoPrint's UI, or both?
> 2. Can you start monitoring, sing a short song like "baby shark", stop monitoring, and then upload ~/.octoprint/logs for me to inspect?
> 
> Thanks!

**Comment by leigh-johnson** [comment]
Oh, thank you for pointing out the log issue! 

I just realized OctoPrint Nanny is logging to octoprint.log, probably because I mucked around with the root logger for multi-process log aggregation in an earlier release.

I'll fix this up so OctoPrint Nanny follows the convention of logging to its own file.

Side note, I love how much functionality Gina has built into OctoPrint. Setting up log handlers automatically is such a great decision. 😍
